### PR TITLE
fix: allow to import dashboards in k8s

### LINKED ISSUE #28: k8s oars init job "Import dashboard failed for an unknown reason"
## Description

After several tests performed, it was found that the init job in k8s for oars was failing to import all the data correctly.

The error is below:

```bash
Collecting superset-api-client<0.7,>=0.6
--
Thu, Apr 27 2023 1:54:29 pm | Downloading superset_api_client-0.6.0-py3-none-any.whl (16 kB)
Thu, Apr 27 2023 1:54:29 pm | Collecting requests-oauthlib
Thu, Apr 27 2023 1:54:29 pm | Downloading requests_oauthlib-1.3.1-py2.py3-none-any.whl (23 kB)
Thu, Apr 27 2023 1:54:29 pm | Collecting requests>=2.0.0
Thu, Apr 27 2023 1:54:29 pm | Downloading requests-2.29.0-py3-none-any.whl (62 kB)
Thu, Apr 27 2023 1:54:29 pm | Collecting pyyaml>=6.0
Thu, Apr 27 2023 1:54:29 pm | Downloading PyYAML-6.0-cp38-cp38-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_12_x86_64.manylinux2010_x86_64.whl (701 kB)
Thu, Apr 27 2023 1:54:29 pm | Collecting charset-normalizer<4,>=2
Thu, Apr 27 2023 1:54:29 pm | Downloading charset_normalizer-3.1.0-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (195 kB)
Thu, Apr 27 2023 1:54:29 pm | Collecting idna<4,>=2.5
Thu, Apr 27 2023 1:54:29 pm | Downloading idna-3.4-py3-none-any.whl (61 kB)
Thu, Apr 27 2023 1:54:29 pm | Collecting urllib3<1.27,>=1.21.1
Thu, Apr 27 2023 1:54:29 pm | Downloading urllib3-1.26.15-py2.py3-none-any.whl (140 kB)
Thu, Apr 27 2023 1:54:29 pm | Collecting certifi>=2017.4.17
Thu, Apr 27 2023 1:54:29 pm | Downloading certifi-2022.12.7-py3-none-any.whl (155 kB)
Thu, Apr 27 2023 1:54:29 pm | Collecting oauthlib>=3.0.0
Thu, Apr 27 2023 1:54:29 pm | Downloading oauthlib-3.2.2-py3-none-any.whl (151 kB)
Thu, Apr 27 2023 1:54:29 pm | Installing collected packages: urllib3, idna, charset-normalizer, certifi, requests, oauthlib, requests-oauthlib, pyyaml, superset-api-client
Thu, Apr 27 2023 1:54:30 pm | Successfully installed certifi-2022.12.7 charset-normalizer-3.1.0 idna-3.4 oauthlib-3.2.2 pyyaml-6.0 requests-2.29.0 requests-oauthlib-1.3.1 superset-api-client-0.6.0 urllib3-1.26.15
Thu, Apr 27 2023 1:54:30 pm | WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
Thu, Apr 27 2023 1:54:30 pm | WARNING: You are using pip version 21.1.3; however, version 23.1.2 is available.
Thu, Apr 27 2023 1:54:30 pm | You should consider upgrading via the '/usr/local/bin/python -m pip install --upgrade pip' command.
Thu, Apr 27 2023 1:54:32 pm | Traceback (most recent call last):
Thu, Apr 27 2023 1:54:32 pm | File "/app/oars/pythonpath/superset-dashboard.py", line 108, in <module>
Thu, Apr 27 2023 1:54:32 pm | update_assets()
Thu, Apr 27 2023 1:54:32 pm | File "/app/oars/pythonpath/superset-dashboard.py", line 54, in update_assets
Thu, Apr 27 2023 1:54:32 pm | getattr(superset, asset_type).import_file(
Thu, Apr 27 2023 1:54:32 pm | File "/usr/local/lib/python3.8/site-packages/supersetapiclient/base.py", line 291, in import_file
Thu, Apr 27 2023 1:54:32 pm | raise_for_status(response)
Thu, Apr 27 2023 1:54:32 pm | File "/usr/local/lib/python3.8/site-packages/supersetapiclient/base.py", line 44, in raise_for_status
Thu, Apr 27 2023 1:54:32 pm | raise ComplexBadRequestError(*e.args, request=e.request, response=e.response, errors=errors) from None
Thu, Apr 27 2023 1:54:32 pm | supersetapiclient.exceptions.ComplexBadRequestError: [
Thu, Apr 27 2023 1:54:32 pm | {
Thu, Apr 27 2023 1:54:32 pm | "message": "Import dashboard failed for an unknown reason",
Thu, Apr 27 2023 1:54:32 pm | "error_type": "GENERIC_COMMAND_ERROR",
Thu, Apr 27 2023 1:54:32 pm | "level": "error",
Thu, Apr 27 2023 1:54:32 pm | "extra": {
Thu, Apr 27 2023 1:54:32 pm | "issue_codes": [
Thu, Apr 27 2023 1:54:32 pm | {
Thu, Apr 27 2023 1:54:32 pm | "code": 1010,
Thu, Apr 27 2023 1:54:32 pm | "message": "Issue 1010 - Superset encountered an error while running a command."
Thu, Apr 27 2023 1:54:32 pm | }
Thu, Apr 27 2023 1:54:32 pm | ]
Thu, Apr 27 2023 1:54:32 pm | }
Thu, Apr 27 2023 1:54:32 pm | }
Thu, Apr 27 2023 1:54:32 pm | ]
```

While superset logs:
[superset logs.txt](https://github.com/openedx/tutor-contrib-oars/files/11347135/superset.logs.txt)

Based on the superset logs, it seems it's related to this bug https://github.com/apache/superset/issues/22506 but it's unclear why it worked on tutor local and not for tutor k8s.

It's also curious that you can run the local python script generated by tutor stored in `env/plugins/oars/apps/pythonpath/superset-dashboard.py` manually, and the dashboard it's imported correctly in you set the following variable into the script:

SUPERSET_DATA_ASSETS_DIR = "env/plugins/oars/apps/data/superset"

Also, you can comment the line that removes the temporary zip file, check for that file in your system (probably in `/tmp`) and import it manually through the superset interface in dashboard. (there should be 4 temporary zip files, search the one that has a folder called `dashboard`)

### PR DESCRIPTION
**Pull request by Ian2012**
## Description

This PR:
- Fixes #28
- Depends https://github.com/openedx/tutor-contrib-superset/pull/45

**Notes:**
- The old dashboards and data are back but are just for testing. I guess once this is approved, I will remove those data as it's outdated.
- Have in mind that the metadata.yaml is pointing to a dashboard, so the import process will be broken till we add at least one proper dashboard

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Ian2012!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.

File: tutoroars/templates/oars/jobs/init/superset-api-dashboard.sh [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
 set -e
 
-pip install -r /app/oars/pythonpath/requirements.txt
-python /app/oars/pythonpath/superset-dashboard.py && echo "Success"
+#
+# Always install local overrides first
+#
+/usr/bin/env bash /app/docker/docker-bootstrap.sh
+
+/usr/bin/env bash /app/docker/docker-bootstrap.sh
```
Comment (bmtcril) [inline tutoroars/templates/oars/jobs/init/superset-api-dashboard.sh:9]: Looks like you're bootstrapping twice here?

**Review by bmtcril — CHANGES_REQUESTED** [review]
I think removing the job patches will also break the `load_xapi_test_data` command?

**Comment by bmtcril** [comment]
Do you think that we could add extensibility by just mounting other directories of assets under the `oars/data` dir? It seems like they might just get picked up, but I'm a little hazy on this process.

File: tutoroars/templates/oars/jobs/init/superset-api-dashboard.sh (hunk 1 above)
Comment (Ian2012) [reply tutoroars/templates/oars/jobs/init/superset-api-dashboard.sh:9]: ups

**Comment by Ian2012** [comment]
It will allow at least to import custom dashboards, I guess we have to get use cases first so that we can verify where to add extensibility

**Pull request merged**